.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
 env:
   global:
     - COMPOSER_ARGS="--no-interaction"
-    - COVERAGE_DEPS="php-coveralls/php-coveralls"
+    - DEPENDENCIES=""
```

Comment:
Technically, we don't need to pre-define globals; the shell will resolve it to `''` if not defined.